utils/devel/rebuild_api.sh

```diff
@@ -29,7 +29,8 @@ cp $gitDirName/openapi/.redocly.yaml .
 npx @redocly/cli lint API.yaml || exit
 
 # Build the documentation
-redoc-cli build API.yaml -o $gitDirName/doc/openapi/LedgerSMB-api.html 
```

Comment:
I had expected to see an update to package.json. why isn't that the case when it says "upgrade"?